packages/terra-button-group/src/ButtonGroup.module.scss

```diff
@@ -70,4 +70,12 @@
     border-bottom-right-radius: var(--terra-button-group-button-border-bottom-right-radius, 0.2142rem);
     border-top-right-radius: var(--terra-button-group-button-border-top-right-radius, 0.2142rem);
   }
+
+  .block {
+    width: 100%;
```

Comment:
Do we need display: block as well here?

Reply:
Hmm, I don't believe so, I think we'll want this to keep the `display: inline-flex` style from `.button-group`.